src/component/main-content.js

```diff
@@ -45,17 +30,21 @@ export default class MainContent extends React.Component {
 
   render() {
 
-    let {requests, config, activeWindow} = this.props;
+    let {
+      requestData,
+      config,
+      activeWindow,
+      setFromIndex,
+      filterRequests
+    } = this.props;
+
     let {activeRequest} = this.state;
     let SearchBar, SetupInstructions;
 
-    const filterRequests = (str) => {
-      this._filterRequests(str);
-      this.render();
-    };
-
     const setActiveRequest = (request) => {
-      this._setActiveRequest(request);
+      this.setState({
+        activeRequest: request
+      });
     };
 
     if(activeRequest) {
```

Comment:
Please turn on http://eslint.org/docs/rules/no-shadow. Looks like "activeRequest" is used for a "request" object and also a "activeRequestComponent".